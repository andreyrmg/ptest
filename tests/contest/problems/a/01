10
2 1001
10 10
10 2
2 1110
10 9
2 110
10 4
2 10
2 1101
10 6
