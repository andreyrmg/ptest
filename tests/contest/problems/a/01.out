2 10
10 2
10 4
2 110
10 6
2 1001
10 9
10 10
2 1101
2 1110
